tensorbay/client/commit_status.py

```diff
@@ -73,21 +75,21 @@ def check_authority_for_commit(self) -> None:
         """Check whether the status is a legal commit.
 
         Raises:
-            TypeError: When the status is not a legal commit.
+            CommitStatusError: When the status is not a legal commit.
 
         """
         if self._draft_number is not None:
-            raise TypeError("The status is not commit")
+            raise CommitStatusError("The status is not commit")
```

Comment:
The input parameter can be the required status, and the error message can be formed in `__str__()`.